lib/smooth/application/io/i2c/BME280.cpp

```diff
@@ -14,15 +14,17 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-
 #include <vector>
-#include <sstream>
 #include <smooth/application/io/i2c/BME280.h>
-#include <smooth/core/util/FixedBuffer.h>
-#include <smooth/core/util/ByteSet.h>
+#include <smooth/core/logging/log.h>
+
+using namespace smooth::core;
+using namespace smooth::core::logging;
 
 namespace smooth::application::sensor
 {
+    static const char TAG[] = "BME280SPI:";
```

Comment:
I believe the ":" is redundant here?

Nitpicking - prefer char*  over [] like in the rest of the code base.